resources/js/processes/modeler/components/inspector/StageList.vue

```diff
@@ -85,13 +80,6 @@ const totalStages = computed(() => stages.value.length);
 const disableButton = computed(() => totalStages.value >= 8);
 const stateNewStage = ref(true);
 
-const onInput = (event) => {
-  // Limit to 200 characters
-  if (newStage.value.length > 200) {
-    newStage.value = newStage.value.substring(0, 200);
-  }
-};
-
```

Comment:
don't remove this validation
The ticket only mentions removing the number of characters in the view, not validating the number of characters.